src/ophyd_async/core/signal.py

```diff
@@ -253,6 +253,34 @@ def set_sim_callback(signal: Signal[T], callback: ReadingValueCallback[T]) -> No
     return _sim_backends[signal].set_callback(callback)
 
 
+def soft_signal_rw(
+    datatype: Optional[Type[T]],
+    name: str,
+    initial_value: Optional[T] = None,
+) -> SignalRW[T]:
+    """Creates a read-writable Signal with a SimSignalBackend"""
+    signal = SignalRW(
+        SimSignalBackend(datatype, initial_value)
+    )
+    signal.set_name(name)
```

Comment:
Please can we make `name=None` be passed to these helpers, and if it isn't already there then add it as an optional arg to `Signal.__init__`

Reply:
And also add this to `epics_signal_*` helpers

Reply:
Ah! Missed this comment somehow, might have to be a new PR.